### ISSUE TITLE
Gene filter not working

### ISSUE THREAD

**Issue by OguzhanB** [rghu/reconCNV#3]
Hi,

This is a great program to analyse CNVs. I am using it for WES CNV analysis. When I select anything from gene filter table nothing appears on filtered data table.

If you need any documents to understand what the problem is, please let me know.

Regards,
Oğuzhan

![Screenshot from 2020-11-19 01-09-47](https://user-images.githubusercontent.com/6258917/99645580-402f2780-2a04-11eb-8c7a-7db812a9f35d.png)

**Comment by rghu** [comment]
Would it be possible for you to share the files used to generate this output (including the config)? I can take a look into this issue for you.

**Comment by OguzhanB** [comment]
> Would it be possible for you to share the files used to generate this output (including the config)? I can take a look into this issue for you.

Hi, I attached all files. I changed file extension tobe able to upload.

[xCNR.txt](https://github.com/rghu/reconCNV/files/5571964/xCNR.txt)
[xCNS.txt](https://github.com/rghu/reconCNV/files/5571965/xCNS.txt)
[XgenemetricsCNS.txt](https://github.com/rghu/reconCNV/files/5571966/XgenemetricsCNS.txt)

[config.txt](https://github.com/rghu/reconCNV/files/5571972/config.txt)
[GRCh38_genome_length_chr23.txt](https://github.com/rghu/reconCNV/files/5571973/GRCh38_genome_length_chr23.txt)

**Comment by rghu** [comment]
That's strange, I'm not able to reproduce the error (see image below). The filtering is performed using javascript so make sure you have the latest version of Chrome web browser installed. Are you seeing any errors when you run reconCNV?

<img width="1175" alt="xCNR_reconCNV_Plot" src="https://user-images.githubusercontent.com/12452200/99819143-b3c05a00-2b14-11eb-91ed-575eaec0ef4d.png">

**Comment by OguzhanB** [comment]
Hi again,

I tried with different browser but nothing changed. I got some error like in the picture. Also I am sending an error in browser.

Oğuzhan

![Screenshot from 2020-11-23 01-56-34](https://user-images.githubusercontent.com/6258917/99949564-19d4f900-2d30-11eb-88dd-0f325c201ee2.png)
![Screenshot from 2020-11-23 02-02-41](https://user-images.githubusercontent.com/6258917/99949569-1c375300-2d30-11eb-9462-1c869088418a.png)
